### DISCUSSION TITLE
Popup的宽度如何调整？

### DISCUSSION THREAD

**Discussion by xmsz-stu** [alibaba-fusion/next#4637] [Q&A]
我有一个组件用到Popup，组件本身的宽度是w-124，但是popup生成的只有192px
<img width="1004" alt="image" src="https://github.com/alibaba-fusion/next/assets/139225486/c001018b-8267-4c51-94e7-95cbc388c9ed">

**Comment by eternalsky** [comment]
Popup 的宽度由内容决定，但也可以通过设置 wrapperClassName  或者 style 来自己指定

**Reply by xmsz-stu** [reply]
那w-124是内容的宽度，为什么外层还是192

**Reply by eternalsky** [reply]
> 那w-124是内容的宽度，为什么外层还是192

这种最好还是有些复现 demo 比较好聊一些
